engine/orchestration/src/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1073,6 +1079,24 @@ public void orchestrateStart(final String vmUuid, final Map<VirtualMachineProfil
                             if (s_logger.isDebugEnabled()) {
                                 s_logger.debug("Start completed for VM " + vm);
                             }
+                            final Host vmHost = _hostDao.findById(destHostId);
```

Comment:
I'll explore this, it's funny because it was actually very huge and I actually have refactored it a lot, and we need cmd-answers needed to do deployments etc, I decided to keep it here for closeness.